### PR TITLE
changed to use code editor

### PR DESCRIPTION
**Pull request by edewit**
use the `<CodeEditor` on the `<JsonFileUpload` component

### CODE REVIEW

File: src/components/json-file-upload/JsonFileUpload.tsx [hunk 1]
```diff
@@ -60,7 +61,7 @@ export const JsonFileUpload = ({
     } else {
       setFileUpload({
         ...fileUpload,
-        value,
+        value: value as string,
```
Comment (jonkoops) [inline src/components/json-file-upload/JsonFileUpload.tsx:64]: Are you sure this value will always be a string?

File: src/components/json-file-upload/JsonFileUpload.tsx [hunk 2]
```diff
@@ -147,7 +148,18 @@ export const JsonFileUpload = ({
             dropzoneProps={{
               accept: ".json",
             }}
-          />
+            hideDefaultPreview
+          >
+            <CodeEditor
+              isLineNumbersVisible
+              code={fileUpload.value as string}
```
Comment (jonkoops) [inline src/components/json-file-upload/JsonFileUpload.tsx:155]: This is already always a string according to the type, so this type cast can be removed.
```suggestion
              code={fileUpload.value}
```

File: src/components/json-file-upload/JsonFileUpload.tsx [hunk 3]
```diff
@@ -147,7 +148,18 @@ export const JsonFileUpload = ({
...
+              language={Language.json}
+              height="128px"
+              onChange={(value, event) =>
```
Comment (jonkoops) [inline src/components/json-file-upload/JsonFileUpload.tsx:158]: Would be nice if we get an actual type for this event, perhaps we can open a bug upstream?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/components/json-file-upload/JsonFileUpload.tsx (hunk 3 above)
Comment (ssilvert) [reply src/components/json-file-upload/JsonFileUpload.tsx:158]: +1

**Review by jonkoops — APPROVED** [review]

**Pull request merged**